utils.js

```diff
@@ -6617,29 +6636,65 @@ export const extractJarArchive = function (
       if (jarResult.status !== 0) {
         console.error(jarResult.stdout, jarResult.stderr);
       } else {
-        if (existsSync(manifestFile)) {
+        let group = "",
+          name = "",
+          version = "",
+          confidence = 1,
+          technique = "manifest-analysis";
+        // When maven descriptor is available take group, name and version from pom.properties
+        // META-INF/maven/${groupId}/${artifactId}/pom.properties
+        // see https://maven.apache.org/shared/maven-archiver/index.html
+        if (existsSync(mavenDir)) {
+          let groupDir = readdirSync(mavenDir);
+          if (groupDir && groupDir.length) {
+            let artifactDir = readdirSync(join(mavenDir, groupDir[0]));
```

Comment:
@Nikemare, This line is failing while creating an SBOM for container images.

```
Error: ENOTDIR: not a directory, scandir '/tmp/jar-deps-pgCSWJ/META-INF/maven/extension.xml'
    at readdirSync (node:fs:1516:26)
    at extractJarArchive (file:///mnt/work/CycloneDX/cdxgen/utils.js:6650:31)
    at createJarBom (file:///mnt/work/CycloneDX/cdxgen/index.js:1084:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async createMultiXBom (file:///mnt/work/CycloneDX/cdxgen/index.js:4868:15)
    at async createBom (file:///mnt/work/CycloneDX/cdxgen/index.js:5388:21)
    at async file:///mnt/work/CycloneDX/cdxgen/bin/cdxgen.js:366:20 {
  errno: -20,
```

Reply:
@prabhu I can provide a fix in the evening (CET), or will you fix it on your own?

Reply:
The evening sounds good! Thank you so much!

Reply:
See #727